packages/flutter/lib/src/material/overscroll_indicator.dart

```diff
@@ -166,6 +171,25 @@ class _OverscrollIndicatorState extends State<OverscrollIndicator> {
     _hide();
   }
 
+  void _refreshHideTimer() {
+    _hideTimer?.cancel();
+    _hideTimer = new Timer(_kIndicatorTimeoutDuration, _hide);
+  }
+
+  bool _isOverscrolling(double scrollOffset) {
```

Comment:
This is just a nit: since you're testing a single value it seems like the method should be called _isOverscroll().